zproject/prod_settings_template.py

```diff
@@ -764,6 +764,15 @@
 ## Controls whether Zulip will rate-limit user requests.
 # RATE_LIMITING = True
 
+## Entries in this dictionary will override Zulip's default rate limits,
+## which are hard-coded in the codebase in RATE_LIMITING_RULES. Rules
+## for domains which are not explicitly overridden here will be as default.
```

Comment:
It might be good to reference where admins can find the set of rules they can override (i.e. `see RATE_LIMITING_RULES in computed_settings.py`), and what the tuples mean.

Reply:
Yeah, makes sense, tweaked to mention the file (default_settings.py, given the adjusted interface)